src/github/client/github-app-client.ts

```diff
@@ -84,8 +85,15 @@ export class GitHubAppClient extends GitHubClient {
 		});
 	};
 
-	public getInstallations = async (): Promise<AxiosResponse<Octokit.AppsGetInstallationResponse>> => {
-		return await this.axios.get<Octokit.AppsGetInstallationResponse>(`/app/installations`, {});
+	public getInstallations = async (): Promise<AxiosResponse<Octokit.AppsGetInstallationResponse[]>> => {
+		return await this.axios.get<Octokit.AppsGetInstallationResponse[]>(`/app/installations`, {});
 	};
 
+	public async searchUserRepositoriesRest(queryString: string): Promise<AxiosResponse<SearchedRepositoriesResponse>> {
```

Comment:
Can we rename this searchAppRepositoriesRest or searchRepositoriesRest

Reply:
Renamed